pkg/policy/policy.go

```diff
@@ -71,9 +72,23 @@ type PolicyType interface {
 var allPolicyTypes = map[string]PolicyType{}
 
 type Store struct {
-	Dir       string
-	Policies  map[PolicyType][]*Policy
-	PolicyIds map[string]string
+	Dir                    string
+	Policies               map[PolicyType][]*Policy
+	PolicyIds              map[string]string
+	SkipPolicyIDResolution bool
```

Comment:
Can you explain the need to make this optional?  I can't find where it's used in the code.

Also, I suggest using the name "validation" instead of "resolution".

Reply:
When running a scan the downloaded policies will have sid/policyId that have been rewritten by the api-server for lacework policies and all custom policies will be vetted to start `c-` as per the design. Thus we don't want the cli building the policyId from the folder name when running a scan or we end up with `c-` policies again for lacework policies. It's set when creating the store after downloading the policy archive and It's used when loading policies 

```

	if m.SkipPolicyIDResolution {
		policy.ID = policy.Metadata.GetString("policyId")
	} else {
		id, err := m.resolvePolicyID(policyType, path)
		if err != nil {
			return nil, err
		}
		policy.ID = id
		policy.Metadata["policyId"] = policy.ID
		policy.Metadata["sid"] = policy.ID
	}
```